app/org/sagebionetworks/bridge/services/backfill/UserConsentBackfill.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.bridge.services.backfill;
+
+import java.util.Iterator;
+
+import org.sagebionetworks.bridge.dao.AccountDao;
+import org.sagebionetworks.bridge.dao.UserConsentDao;
+import org.sagebionetworks.bridge.models.accounts.Account;
+import org.sagebionetworks.bridge.models.accounts.HealthId;
+import org.sagebionetworks.bridge.models.backfill.BackfillTask;
+import org.sagebionetworks.bridge.services.HealthCodeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component("userConsentEventBackfill")
+public class UserConsentBackfill extends AsyncBackfillTemplate {
+
+    private AccountDao accountDao;
+    private UserConsentDao userConsentDao;
+    private HealthCodeService healthCodeService;
+    
+    @Autowired
+    private final void setAccountDao(AccountDao accountDao) {
+        this.accountDao = accountDao;
+    }
+    @Autowired
+    private final void setUserConsentDao(UserConsentDao userConsentDao) {
+        this.userConsentDao = userConsentDao;
+    }
+    @Autowired
+    public void setHealthCodeService(HealthCodeService healthCodeService) {
```

Comment:
Nitpick: final. We're unlikely to actually override this, but that one time we did, and bad things happened.